include/zephyr/dt-bindings/gpio/numicro-gpio.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2022 SEAL AG
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_GPIO_NUVOTON_NUMICRO_GPIO_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_GPIO_NUVOTON_NUMICRO_GPIO_H_
+
+/**
+ * @brief Enable GPIO pin debounce.
+ *
+ * The debounce flag is a Zephyr specific extension of the standard GPIO flags
+ * specified by the Linux GPIO binding. Only applicable for Nuvoton NuMicro SoCs.
+ */
+#define NUMICRO_GPIO_INPUT_DEBOUNCE	(1U << 8)
+
+#define NUMICRO_GPIO_SLEW_RATE_NORMAL	(0U << NUMICRO_GPIO_SLEW_RATE_SHIFT)
+#define NUMICRO_GPIO_SLEW_RATE_HIGH	(1U << NUMICRO_GPIO_SLEW_RATE_SHIFT)
+#define NUMICRO_GPIO_SLEW_RATE_FAST	(2U << NUMICRO_GPIO_SLEW_RATE_SHIFT)
+#define NUMICRO_GPIO_SLEW_RATE_MASK	(3U << NUMICRO_GPIO_SLEW_RATE_SHIFT)
+#define NUMICRO_GPIO_SLEW_RATE_SHIFT	9
```

Comment:
looks like this is really meaningful in pinctrl context, will GPIO API (slow) ever care about slew rate?

Reply:
Probably not, will move it to the pinctrl driver